tests/oss-fuzz/build.sh

```diff
@@ -36,20 +36,53 @@
 #     avif_fuzztest_enc_dec_incr@EncodeDecodeAvifFuzzTest.EncodeDecodeGridValid \
 #     --sanitizer address
 
-# Reset compile flags to build dav1d without fuzzer flags. The meson build system
-# is problematic with sanitizer flags.
+# Build dav1d with sanitizer flags.
+# This duplicates dav1d.cmd but includes extra flags: -Db_sanitize=$SANITIZER -Db_lundef=false,
+# and -Denable_asm=false for msan
+cd ext
+git clone -b 1.4.0 --depth 1 https://code.videolan.org/videolan/dav1d.git
```

Comment:
We could also apply a patch to `dav1d.cmd` before running it, to keep most of the instructions, including the version. We would not forget to update it here too.  
However this is a temporary solution if I understood correctly so it is not worth the hassle.

Reply:
Indeed, that is temporary until https://github.com/AOMediaCodec/libavif/pull/1744 but let's do it anyway.